pkgs/development/python-modules/multivolumefile/default.nix

```
@@ -0,0 +1,36 @@
+{
+  buildPythonPackage,
+  fetchPypi,
+  lib,
+  nix-update-script,
+  python3Packages,
+  pythonOlder,
+}:
+
+buildPythonPackage rec {
+  pname = "multivolumefile";
+  version = "0.2.3";
+  pyproject = true;
+
+  disabled = pythonOlder "3.6";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-oGSNCq+8luWRmNXBfprK1+tTGr6lEDXQjOgGDcrXCdY=";
+  };
+
+  build-system = with python3Packages; [
```

```suggestion
  build-system = [
```
Same for all other packages.

---

Fixed